docs/examples/index.md

```diff
@@ -15,8 +16,29 @@ Check out these examples of different editor integrations. See the predefined bu
 </info-box>
 </span>
 
+## Predefined builds
+
+{@link installation/getting-started/predefined-builds Predefined CKEditor&nbsp;5 builds} are a set of ready-to-use rich text editors. Every “build” provides a single type of editor with a set of features and a default configuration. They provide convenient solutions that can be installed with no effort and that satisfy the most common editing use cases. See all of them in action: {@link examples/builds/classic-editor classic}, {@link examples/builds/inline-editor inline}, {@link examples/builds/balloon-editor balloon}, {@link examples/builds/balloon-block-editor balloon block}, {@link examples/builds/document-editor document}, and {@link examples/builds/multi-root-editor multi-root} editors! Learn their differences by using our examples and analyzing the source code of the builds.
```

Comment:
> They provide convenient solutions that can be installed with no effort and that satisfy the most common editing use cases.

This sentence sounds complex to me. Maybe we could avoid the passive voice, like: "They cover most common editing use cases, and you can install them easily."